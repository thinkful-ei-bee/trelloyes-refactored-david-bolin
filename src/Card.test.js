import React from 'react';
import ReactDOM from 'react-dom';
import Card from './Card.js';
import renderer from 'react-test-renderer';

describe('Card Component', () => {
  it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<Card title="x" content="z"/>, div);
    ReactDOM.unmountComponentAtNode(div);
  });

  it('renders the UI as expected', () => {
    const tree = renderer
      .create(<Card title="First Card" content="lorem ipsum" />)
      .toJSON();
    expect(tree).toMatchSnapshot();  
    });
});
