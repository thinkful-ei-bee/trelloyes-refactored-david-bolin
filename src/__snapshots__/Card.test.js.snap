// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card Component renders the UI as expected 1`] = `
<div
  className="Card"
>
  <button
    onClick={[Function]}
    type="button"
  >
    delete
  </button>
  <h3>
    First Card
  </h3>
  <p>
    lorem ipsum
  </p>
</div>
`;
